web/package.json

```diff
@@ -14,12 +14,15 @@
     "build:malte": "webpack --config tools/webpack.config.babel.js --env.config_name=malte",
     "build:integreat-test-cms": "webpack --config tools/webpack.config.babel.js --env.config_name=integreat-test-cms",
     "check:build": "yarn es-check es5 ./dist/**/*.js --verbose",
-    "test": "jest --config jest.config.json",
-    "test:coverage": "jest --config jest.config.json --coverage",
-    "test:watch": "jest --config jest.config.json --watchAll",
-    "test:update": "jest --config jest.config.json -u",
-    "lint": "eslint . && yarn run stylelint",
+    "test": "jest --config jest.config.js",
+    "test:coverage": "jest --config jest.config.js --coverage",
+    "test:ci": "yarn run test:coverage --ci",
```

Comment:
what does the `--ci` do?

Reply:
Disables generation of snapshots and throws an error afaik